replica/distributed_loader.cc

```diff
@@ -580,16 +579,20 @@ future<> table_population_metadata::start_subdir(sstring subdir) {
     _highest_generation = std::max(generation, _highest_generation);
 }
 
+sstables::shared_sstable make_sstable(replica::column_family& table, fs::path dir, sstables::generation_type generation, sstables::sstable_version_types v) {
+    return table.get_sstables_manager().make_sstable(table.schema(), dir.native(), generation, v, sstables::sstable_format_types::big);
```

Comment:
the idea is to eventually kill the name column_family which was aliased to table. so please use replica::table instead.